spec/spec_helper.rb

```diff
@@ -93,4 +93,7 @@
   # as the one that triggered the failure.
   Kernel.srand config.seed
 =end
+  config.before(:suite) do
```

Comment:
if something happens (e.g. I run ctrl+c) I'll be left out with records in the testing DB, right? (might be worth checking just in case it changed with Rails 6)

If yes, it'd be good to find a way to clear the DB first thing when running tests (even before loading seeds)